python/src/scipp/compat/mantid.py

```diff
@@ -40,8 +39,8 @@ def init_pos_spectrum_no(nHist, ws):
         else:
             pos[i, :] = [np.nan, np.nan, np.nan]
         num[i] = ws.getSpectrum(i).getSpectrumNo()
-    pos = sc.Variable([sc.Dim.Position], values=pos, unit=sc.units.m)
-    num = sc.Variable([sc.Dim.Position], values=num)
+    pos = sc.Variable([sc.Dim.Spectrum], values=pos, unit=sc.units.m)
```

Comment:
Don't we now need a `dtype=vector_3d_double` or something like that to get eigen vectors for the position?

Reply:
I had tried and it worked (but please double check). I think not in this case, because we are passing a 2d array, not a 1d list. We may need to work on making this more consistent.

Reply:
Igor is working on this. Something also needed to be done in the bind 1d when you added the support for pyobject. He has a fix